apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -18,3 +18,47 @@ describe('When: I use the reading list feature', () => {
     );
   });
 });
+
+
+describe('When: I use the want to read to see the snack bar', () => {
+  it('Then: I should see snackbar', async () => {
+    await browser.get('/');
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
+    );
+
+    const form = await $('form');
```

Comment:
FYI many of these awaits are not needed, check out: https://www.protractortest.org/#/async-await